Updating known issues for 3.1.102 SDK (#4293)
dotnet/core: release-notes/3.1/3.1-known-issues.md

Visual Studio 16.4 Preview 2 may uninstall .NET Core 3.0 when installed with the .NET Core 3.1 SDK. We recommend you re-install or repair the [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) if this happens.

### .NET Core 3.1.2 SDK (3.1.102 SDK)

> Packages built out of dotnet/corefx but part of the "Microsoft.AspNetCore.App" Shared Framework doesn't exist in the targeting pack 3.1.2. Due to this after upgrading to 3.1.2 apps which were able to compile against some assemblies can not longer compile. This will have an impact on both .NET Core as well as ASP.NET Core.

Reviewer: ```suggestion
Packages built out of dotnet/corefx but part of the "Microsoft.AspNetCore.App" Shared Framework don't exist in the targeting pack for 3.1.2. Due to this, after upgrading to 3.1.2, apps which were able to compile against some assemblies can not longer compile. This will have an impact on both .NET Core as well as ASP.NET Core.
```

> Packages built out of dotnet/corefx but part of the "Microsoft.AspNetCore.App" Shared Framework don't exist in the targeting pack for 3.1.2. Due to this, after upgrading to 3.1.2, apps which were able to compile against some assemblies can not longer compile. This will have an impact on both .NET Core as well as ASP.NET Core.

Reviewer 2: Make it "... can no longer compile ..."

List of affected assemblies 
* System.Diagnostics.EventLog
* System.IO.Pipelines
* System.Security.AccessControl
* System.Security.Cryptography.Cng
* System.Security.Cryptography.Xml
* System.Security.Permissions
* System.Security.Principal.Windows
* System.Windows.Extensions

Current workaround for this issue is to add an explicit PackageReference to the packages corresponding to above mentioned assemblies. 

> You can find more information about this issue on [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/issues/19133 )

Reviewer 2: ```suggestion
You can find more information about this issue on [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/issues/19133)
```

> A fix for this should be available in later release of 3.1.

Reviewer 2: ```suggestion
A fix for this will be available in a future servicing release of 3.1.X.
```
